### PR TITLE
fix(color-picker): Hide hash sign for accessibility

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/364
It now announces `Label, edit text, FFFFFF`
## Checklist

- [ ] branch has been rebased on the latest master commit
- [ ] tests are changed or added
- [ ] `yarn test` passes
- [ ] all (dev)dependencies that the module needs is added to its `package.json`
- [ ] code has been documented and, if applicable, usage described in README.md
- [ ] module has been added to `canvas-kit-react` and/or `canvas-kit-css` universal modules, if
      applicable
- [ ] design approved final implementation
- [ ] a11y approved final implementation
- [ ] code adheres to the [API & Pattern guidelines](../API_PATTERN_GUIDELINES.md)

### CODE REVIEW

**Comment by anicholls** [comment]
@mannycarrera4 I think this only partially fixes the issue. I left a comment on the issue for @jpante as the language there is quite confusing.

File: modules/color-picker/react/lib/ColorInput.tsx [hunk 1]
```diff
@@ -177,7 +177,9 @@ export default class ColorInput extends React.Component<ColorInputProps> {
             icon={checkSmallIcon}
           />
         ) : null}
-        <PoundSignPrefix disabled={disabled}>#</PoundSignPrefix>
+        <PoundSignPrefix aria-hidden={true} disabled={disabled}>
```
Comment (NicholasBoll) [inline modules/color-picker/react/lib/ColorInput.tsx]: Not sure if this is a part of this PR, but I had to look up what `PoundSignPrefix` was... (part of the problem with the Styled component API). Turns out this is a `span` element which doesn't have a `disabled` attribute. What is the `disabled={true}` supposed to do?

Comment (NicholasBoll) [reply modules/color-picker/react/lib/ColorInput.tsx]: Oh, I see. It is for styling only: https://github.com/Workday/canvas-kit/pull/523/files#diff-f2800e7c1a68cd10843f1a74fb5e1e48R94-R97

Comment (mannycarrera4) [reply modules/color-picker/react/lib/ColorInput.tsx]: it changes the color of the pound sign... which in no way is that clear.

**Comment by anicholls** [comment]
Closing until we have more insight on https://github.com/Workday/canvas-kit/issues/364

**Pull request closed without merging**